### DISCUSSION TITLE
Manual Control

### DISCUSSION THREAD

**Discussion by pettertho** [basnijholt/adaptive-lighting#186] [Q&A] [answered]
The manual control attribute within Home Assistant, what value(s) should I expect when a light has been controlled manually?
If controlling multiple lights with the same adaptive lighting, will it display all manually controlled lights?

**Comment by basnijholt** [comment, marked as answer]
You are correct!

As example:
![image](https://user-images.githubusercontent.com/6897215/133114870-7dde41c9-7131-4026-8625-14ed4595b203.png)

**Comment by pettertho** [comment]
Thanks for confirming! Would be nice to add this information and example values to the documentation/readme if it makes sense to do that.

I have had some issues where manual control was not consistently activating when a light was indeed manually controlled, and could never see a value in `manual_control` field, but did not know what value to expect, if any at all. This was happening with a single adaptive light switch that controlled multiple individual lights. I could never see a value in the attribute.

Maybe a false value if none of the light(s) within a switch are under manual control would make sense?

Edit: Maybe split up this paragraph a bit and mention that adaptive lighting will list any light that is under manual control: https://github.com/basnijholt/adaptive-lighting#taking-back-control
